crates/optimism/consensus/src/lib.rs

```diff
@@ -112,8 +112,16 @@ impl Consensus for OptimismBeaconConsensus {
         Ok(())
     }
 
+    /// Validate a block without regard for state:
+    ///
+    /// - Compares the ommer hash in the block header to the block body
+    /// - Compares the transactions root in the block header to the block body
     fn validate_block_pre_execution(&self, block: &SealedBlock) -> Result<(), ConsensusError> {
-        validate_block_pre_execution(block, &self.chain_spec)
+        // Even though there's no ommers in optimism, `ommer_hash` is still used to calculate the
+        // header hash.
+        validate_ommer_hash(block)?;
+
+        validate_transaction_root(block)
```

Comment:
does no longer checks the withdrawals/requests etc?

this is problematic, because we need to check this because it's part of the block, same in op-geth